subfunctions/demo_CaBBI_FHN

```diff
@@ -0,0 +1,236 @@
+priors.SigmaX0(1,1) =  4;
+priors.SigmaX0(2,2) =  1;
+priors.SigmaX0(3,3) =  25;
+
+% prior densities for evolution parameters
+priors.muTheta         = [0; 0; 0];  
+priors.SigmaTheta(1,1) = 5;                                                % for X_Tau_Ca
+priors.SigmaTheta(2,2) = 100;                                              % for Ca_base 
+priors.SigmaTheta(3,3) = 5;                                                % for X_k_Ca 
+
+% prior densities for observation parameters
+priors.muPhi         = [min(Fluor_trace)/2;0];
+priors.SigmaPhi(1,1) = 0.25;                                               % for d_F
+priors.SigmaPhi(2,2) = 1;                                                  % for X_k_F
+
+% prior densities for precision parameters: Gamma distributions
+priors.a_alpha = 1e0;
+priors.b_alpha = 1e0;
+priors.a_sigma = 1e0;
+priors.b_sigma = 1e0;
+
+% building the precison matrix for evolution variables
+priors.iQx = cell(nFrames,1);
+for t = 1:nFrames
+    dq(1)           = (2);                                                 % on V
+    dq(2)           = (30);                                                % on W
+    dq(3)           = 10;                                                  % on [Ca2+]
+    priors.iQx{t}   = diag(dq);
+end
+
+% building the precision verctor the fluorescnce observations    
+priors.iQy = cell(nFrames,1);
+for t=1:nFrames
+    priors.iQy{t} = 3;                                                     % on F
+end
+
+% setting the option structure
+options.inF     = inF;
+options.inG     = inG;
+options.priors  = priors;
+
+% assigning the dimensions for parameter space  
+dim.n       = 3;                                                           % number of neuroal variables
+dim.n_phi   = 2;                                                           % number of observation paramters
+dim.n_theta = 3;                                                           % number of evolution parameters
+
+
+%% Step 8. Inversion: inverting the FHN model for the given fluorescence trace
+[posterior, out] = VBA_NLStateSpaceModel(Fluor_trace,zeros(1,numel(Fluor_trace)),f_fname,g_fname,dim,options);
+
+
+%% Post-processing
+%% Step 9. Extracting the estimated Voltage and [Ca2+] traces (i.e. the posterior means)
+
+V_inferred = posterior.muX(1,1:end);                                       % inferred Voltage (posterior mean)
+W_inferred = posterior.muX(2,1:end);                                       % inferred recovery variable (posterior mean)
+Ca_inferred = posterior.muX(end,1:end);                                    % inferred [Ca2+] kinetics (posterior mean)
+
+% Plotting the estimated voltage and [Ca2+] traces
+Calcium_basal = 50;                                                        % in [nM], the basal [Ca2+] concentration  
+
+% Plot the pre-processed fluorescence trace
+figure
+subplot(411); plot(1:nFrames, Fluor_trace,'g','LineWidth',2)
+legend('fluoresnce')
```

Comment:
Typo again. I suspect a copy-paste issue. Use find and replace to batch correct.